docs/reference/cache.rst

```diff
@@ -66,6 +66,38 @@ or you can use the `Response` object:
         return Response::create(sprintf("your name is %s", $user->getUsername()))->setTtl(0)->setPrivate();
     }
 
+Cache Keys
+::::::::::
+
+The ``SonataCacheBundle`` needs cache keys in order to find cached blocks.
+By default cache keys consist of the ``block_id`` and the ``updated_at``
+timestamp. Because these values change on every call from the Twig Helper,
+it is mandatory to overwrite the ``getCacheKeys`` function in your custom block class:
+
+.. code-block:: php
+
+    <?php
+
+    namespace App\Block;
+    
+    use Sonata\BlockBundle\Block\Service\AbstractBlockService;
+
+    final class CachedBlock extends AbstractBlockService
+    {
+        public function execute(BlockContextInterface $blockContext, Response $response = null): Response
```

Comment:
Please add a use statement for Response 

Afterwards we are good to go 💪🏻

Reply:
Alright, done :)
Thank you for your suggestions!